h/accounts/views.py

```diff
@@ -124,7 +124,7 @@ def login(self):
 
         self.request.registry.notify(LoginEvent(self.request, user))
 
-        userid = 'acct:{}@{}'.format(user.username, self.request.domain)
+        userid = u'acct:{}@{}'.format(user.username, self.request.domain)
```

Comment:
Yeah I think it is. I noticed a "non-unicode bind param for unicode column" warning from sqlalchemy, and I had changed the type of the username column, so thought it was my fault and fixed it. Looking at the diff though the column was already unicode so the warning should already have been there. We may as well fix it though! It's in its own commit.